interface/billing/sl_receipts_report.php

```diff
@@ -413,38 +431,50 @@ function sel_diagnosis() {
       // If a doctor was specified.
       if ($form_doctor) $query .= " AND u.id = '$form_doctor'";
       ***************************************************************/
+      $sqlBindArray = array();
       $query = "SELECT a.pid, a.encounter, a.post_time, a.code, a.modifier, a.pay_amount, " .
         "fe.date, fe.id AS trans_id, fe.provider_id AS docid, fe.invoice_refno, s.deposit_date, s.payer_id, " .
-        "b.provider_id " .
+        "b.provider_id, concat(lname, ' ', fname) as 'fulname', lname as 'last', fname as 'first' " .
```

Comment:
What tables are these items coming from? Better to prefix them with the table and also better to more descriptive variables as prov_fulname etc.